rerun_py/src/python_bridge.rs

```diff
@@ -2242,3 +2252,112 @@ authkey = multiprocessing.current_process().authkey
     })
     .map(|authkey: Bound<'_, PyBytes>| authkey.as_bytes().to_vec())
 }
+
+#[pyclass(name = "OauthLoginFlow", module = "rerun_bindings.rerun_bindings")] // NOLINT: ignore[py-cls-eq] non-trivial implementation
+struct PyOauthLoginFlow {
+    login_flow: OauthLoginFlow,
+}
+
+#[pymethods] // NOLINT: ignore[py-mthd-str]
+impl PyOauthLoginFlow {
+    /// Get the URL for the OAuth login flow.
+    fn login_url(&self) -> String {
+        self.login_flow.server.get_login_url().to_owned()
+    }
+
+    /// Finish the OAuth login flow.
+    ///
+    /// Returns
+    /// -------
+    /// str
+    ///     The access token.
+    fn finish_login_flow(&mut self) -> PyResult<String> {
+        let result: Result<Credentials, re_auth::callback_server::Error> = Runtime::new()?
```

Comment:
This should be using `wait_for_future`, which re-uses any existing tokio runtime